repos/system_upgrade/el7toel8/libraries/repofileutils.py

```diff
@@ -35,12 +35,26 @@ def parse_repofile(repofile):
     return RepositoryFile(file=repofile, data=data)
 
 
+def get_repodirs():
+    """
+    Return all directories yum scans for repository files.
+    By default, their paths on RHEL 7 should be:
+    ['/etc/yum.repos.d', '/etc/yum/repos.d', '/etc/distro.repos.d']
+
+    ATTENTION: Module dnf is reuired. The paths may not exist.
```

Comment:
spell in required

Reply:
@drehak the current implementation rerurns the paths just related to the HOST system. It ignores the `context`. No blocker for me, as in the current implementation of IPU it should return the same results inside containers as well. But put there FIXME or TODO comment that the content should be get in the used context. So the correct solution would be a utility (or python -c .... ) cmd, that will be executed in the given context and print the list to the output in suitable format (e.g. directory per line, or json).